### ISSUE TITLE
MetaTest

### ISSUE THREAD

**Issue by Yash-Vardhan-Sharma** [tristandeleu/pytorch-meta#59]
Hi,
Thanks for this Library 

How are we supposed to do meta testing because accuracy u mentioned is of meta train episodes, right?

**Comment by tristandeleu** [comment]
Hi! I think you are referring to the examples, where the accuracy is indeed always on meta-training episodes. The examples are meant to be very simple to showcase the library, and they are not meant to have complete features (e.g. evaluation on meta-test, different choices of datasets).

To have a complete (and better tested) implementation of an algorithm based on Torchmeta, I suggest you to look at [pytorch-maml](https://github.com/tristandeleu/pytorch-maml), which does have a separate [`test.py`](https://github.com/tristandeleu/pytorch-maml/blob/master/test.py) script for meta-testing.

**Comment by Yash-Vardhan-Sharma** [comment]
yea , i am trying to implement prototypical nets ,i am finding pytorch-meta's functions to be different , how do i directly do it while loading the data in my protonet code .

**Comment by tristandeleu** [comment]
An option is to follow the [Prototypical Network example](https://github.com/tristandeleu/pytorch-meta/tree/master/examples/protonet) for the meta-training part, and then adapt the `test.py` script from pytorch-maml I linked above for meta-testing.

**Comment by Yash-Vardhan-Sharma** [comment]
Thanks
